commit 586b0ed81828a443717b018989f2bced1bcfac01
Author: Cole Robinson <crobinso@redhat.com>
Date:   Thu Dec 5 14:59:05 2013 -0500

    qemu: hotplug: Fix adding USB devices to the driver list
    
    We were unconditionally removing the device from the host list, when it
    should only be done on error.
    
    This fixes USB collision detection when hotplugging the same device to
    two guests.

Index: libvirt-1.1.1/src/qemu/qemu_hotplug.c
===================================================================
--- libvirt-1.1.1.orig/src/qemu/qemu_hotplug.c	2014-03-25 14:15:28.818627590 -0500
+++ libvirt-1.1.1/src/qemu/qemu_hotplug.c	2014-03-25 14:15:28.810627590 -0500
@@ -1346,9 +1346,9 @@ cleanup:
             virSecurityManagerRestoreHostdevLabel(driver->securityManager,
                                                   vm->def, hostdev, NULL) < 0)
             VIR_WARN("Unable to restore host device labelling on hotplug fail");
+        if (added)
+            virUSBDeviceListSteal(driver->activeUsbHostdevs, usb);
     }
-    if (added)
-        virUSBDeviceListSteal(driver->activeUsbHostdevs, usb);
     if (list && usb &&
         !virUSBDeviceListFind(list, usb) &&
         !virUSBDeviceListFind(driver->activeUsbHostdevs, usb))
